pkg/apis/pxc/v1/pxc_types.go

```diff
@@ -69,6 +69,7 @@ type PXCScheduledBackup struct {
 	ServiceAccountName string                        `json:"serviceAccountName,omitempty"`
 	Annotations        map[string]string             `json:"annotations,omitempty"`
 	PITR               PITRSpec                      `json:"pitr,omitempty"`
+	RuntimeClassName   *string                       `json:"runtimeClassName,omitempty"`
```

Comment:
NodeSelector and RuntimeClassName are very connected things.
so, both items should be set on the same storage level.
it is needed to modify `BackupStorageSpec` (not `PXCScheduledBackup`).

Reply:
It's already rebased onto the main branch. My previous commit is the latest one from the main branch (f512d21ea3b709b8e81bd53ee98d2464f70e0a8d). Did I miss something?

Reply:
Moved RuntimeClassName to BackupStorageSpec